CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,126 @@
+# Contributor Covenant Code of Conduct
```

Comment:
This document has been removed from `epiparameter` in favour of the Epiverse code of conduct. See `epiparameter` README for link.

Reply:
Removed in favour of the Epiverse doc.